openmc/mgxs_library.py

```diff
@@ -744,7 +753,10 @@ def set_nu_fission_mgxs(self, nu_fission, nuclide='total',
             msg = 'Angular-Dependent MGXS have not yet been implemented'
             raise ValueError(msg)
 
-        self._use_chi = True
+        if type(nu_fission) is openmc.mgxs.NuFissionMatrixXS:
```

Comment:
I think the more Pythonic version of this would be to use `isinstance`, but I don't know of a strict rule either way.